src/database/interface.ts

```diff
@@ -70,7 +70,7 @@ export interface DatabaseTournament {
 export interface SynchroniseTournament {
 	name: string;
 	description: string;
-	players: number[];
+	players: { challongeId: number; discordId: string }[];
```

Comment:
Is this value actually completely arbitrary? I thought it was passed to the API

Reply:
the discordId is stored on the challonge API solely for the sake of mapping discord users to players, for players who aren't discord users it can be arbitrary